lucida/template/cpp/Makefile

```diff
@@ -0,0 +1,22 @@
+all:
+	cd server ; \
+	make all ; \
+	cd ../test; \
+	make all ; \
+	cd ..;
+
+start_server: 
+	cd server; ./template_server 
```

Comment:
why not have a global variable that populates this with something useful $MY_SERVICE_NAME. think bigger!